README.md

```diff
@@ -21,6 +21,7 @@ Tested on Ubuntu 18.04 and Debian 10.
 
 * Put all these files in `/var/www/html`
 * Run `sudo setup.sh`
+* Create a daily cron job as to run `deduplicate.sh` using the same account as your webserver
```

Comment:
The command you added to setup.sh already does this, right? I wouldn't mention it here.

Reply:
Oh yeah, I fixed it later